app/command_mute_test.go

```diff
@@ -99,7 +99,6 @@ func TestMuteCommandSpecificChannel(t *testing.T) {
 		UserId:    th.BasicUser.Id,
 	}, channel2.Name)
 	assert.Equal(t, "api.command_mute.success_mute", resp.Text)
-	time.Sleep(time.Millisecond)
```

Comment:
Sleep no longer required since `ToggleMuteChannel` is synchronous with respect to updating the store.